central/graphql/resolvers/components_v2.go

```diff
@@ -197,15 +201,15 @@ func (eicr *imageComponentResolver) TopVuln(ctx context.Context) (VulnerabilityR
 // TopNodeVulnerability returns the first node component vulnerability with the top CVSS score.
 func (eicr *imageComponentResolver) TopNodeVulnerability(ctx context.Context) (NodeVulnerabilityResolver, error) {
```

Comment:
Yeah, that was confusing because in legacy implementation `storage.ImageComponent` was used for both image and node components. I moved the implementation to `nodeComponentResolver` and added conversion methods to convert `storage.ImageComponent` to `storage.NodeComponent` .